### ISSUE TITLE
examples里面的如何恢复训练

### ISSUE THREAD

**Issue by monkeycc** [PaddlePaddle/PaddleSpeech#527]
examples 目录源码
bash ./local/run_train.sh

-------------------------
-# train model
-# if you wish to resume from an exists model, uncomment --init_from_pretrained_model
export FLAGS_sync_nccl_allreduce=0
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \
python3 -u ${MAIN_ROOT}/train.py \
--batch_size=30 \
--num_epoch=50 \
--num_conv_layers=2 \
--num_rnn_layers=3 \
--rnn_layer_size=1024 \
--num_iter_print=100 \
--save_epoch=1 \
--num_samples=120000 \
--learning_rate=5e-4 \
--max_duration=27.0 \
--min_duration=0.0 \
--test_off=False \
--use_sortagrad=True \
--use_gru=True \
--use_gpu=True \
--is_local=True \
--share_rnn_weights=False \
--train_manifest="data/manifest.train" \
--dev_manifest="data/manifest.dev" \
--mean_std_path="data/mean_std.npz" \
--vocab_path="data/vocab.txt" \
--output_model_dir="./checkpoints" \
--augment_conf_path="${MAIN_ROOT}/conf/augmentation.config" \
--specgram_type="linear" \
--shuffle_method="batch_shuffle_clipped" \

-------------
看了教程
`python3 train.py \
--init_from_pretrained_model CHECKPOINT_PATH_TO_RESUME_FROM`

那么这个
--init_from_pretrained_model CHECKPOINT_PATH_TO_RESUME_FROM

这个CHECKPOINT_PATH_TO_RESUME_FROM路径
应该指的是哪里

./local/run_train.sh
直接在代码增加吗
--init_from_pretrained_model = “？？？？？？？”

**Comment by zh794390558** [comment]
参见 https://github.com/PaddlePaddle/DeepSpeech/blob/b3c728d46ff4eee68c45f20b0abb76e968008bcb/model_utils/model.py#L305 
， 这类问题可以多看下代码哈。
